contents/handbook/people/compensation.mdx

```diff
@@ -69,11 +72,23 @@ As we do these much more frequently than regular companies, team members should
 
 ### Pay review process
 
+Pay at PostHog is reviewed much more often at most other companies - at least 2 to 3 times a year.
```

Comment:
I think the time range here could be more specific. It seems that we specify a 3 month cadence for compensation reviews but not for compensation reviews. If this is a conscious decision, an explanation about why we chose this timeline could be helpful for our employees and new hires.

Reply:
Agreed - I've clarified that we do this every quarter now, but that we may change it in future if it's not practical to keep this up (and of course any changes should be documented in a PR).

Reply:
I think James and I are in two minds about this. Doing compensation review genuinely takes us about 15-20 hours each, every time, and we're trying to figure out if 4x year is better than 3x year to justify that amount of time spent. Happy to leave for now but I think we will go down to 3x a year.